src/services/ProblemService.js

```diff
@@ -19,6 +19,10 @@ export class ProblemService extends AbstractService {
         return this.axios.patch(`/api/problems/${problemId}`, {problemId, description});
     }
 
+    async modifyProblemVisible(problemId, visible) {
+        return this.axios.patch(`/api/problems/${problemId}`, {problemId, visible});
+    }
+
```

Comment:
Please rename the following methods:
`modifyProblemTitle` to `updateProblemTitle`
`modifyProblemDescription` to `updateProblemDescription`
`modifyProblemVisible` to `updateProblemVisible`